src/main/target/REVO/target.h

```diff
@@ -136,7 +136,7 @@
 #define MPU_INT_EXTI            PC4
 #define USE_MPU_DATA_READY_SIGNAL
 
-#if !defined(AIRBOTF4) && !defined(REVOLT) && !defined(SOULF4) && !defined(PODIUMF4)
+#if defined(AIRBOTF4) || defined(AIRBOTF4SD)
```

Comment:
@TheAngularity: Little side note: MAG is normally connected through I2C, because that's how the chips are designed. Yet there are almost no boards with on-board MAG chip, since the sensitivity of the chips to electrical noise requires them to be mounted some distance away from the rest of the electronics (often they are combined with GPS modules and mounted on top of a 'mast'). This is why there are a lot of target definitions out there that include MAG, even though the boards do not have a MAG chip on board.